src/compiler/JSPureTemplate.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-disable */
+
+module.exports = function main(runtime) {
+  const $ = {
+    col: runtime.col,
+    exec: runtime.exec.bind(runtime),
+    xss: runtime.xss.bind(runtime),
+    listInfo: runtime.listInfo.bind(runtime),
+    use: runtime.use.bind(runtime),
+    slyResource: runtime.resource.bind(runtime),
+    call: runtime.call.bind(runtime),
+    template: runtime.template.bind(runtime),
+    dom: runtime.dom,
+  };
```

Comment:
would it be possible to replace this part with `const $ = runtime` or `const $ = runtime.templateHelpers()`?

Reply:
I guess :-)

Reply:
let's tackle this in a separate PR